pkg/controllers/user/logging/utils/wrap.go

```diff
@@ -60,13 +60,10 @@ type WrapSyslog struct {
 }
 
 func (w *WrapClusterLogging) Validate() error {
-	curtg, _, _, _, _, err := getWrapConfig(w.ElasticsearchConfig, w.SplunkConfig, w.SyslogConfig, w.KafkaConfig)
+	curtg, _, _, _, _, _, err := getWrapConfig(w.ElasticsearchConfig, w.SplunkConfig, w.SyslogConfig, w.KafkaConfig, w.EmbeddedConfig)
```

Comment:
we should consider revising getWrapConfig action - it seems to return way too many arguments. May be it would make sense to wrap them in the same internal struct